.npmignore

```diff
@@ -6,12 +6,17 @@
 .vagrant
 report.tap
 build/
+coverage/
 demos/
 examples/
+instrumented/
 products/
+reports/
 tests/component-tests/
+tests/configs/
```

Comment:
I added this in the last pull and it absolutely should not be excluded.  It includes the configuration for the test instrumentation.